scripts/LCD-007.py

```diff
@@ -0,0 +1,21 @@
+import requests, json
+BASE = "https://api.openweathermap.org/data/2.5/weather?"
+APIKEY = "fc2822e01a0efcce52cb56212fe44fd5"
```

Comment:
General comment, you don't need to take action on this. It is bad practice to put API keys in the code. Let's say you are working with sensitive data or have a certain quota to the amt of times you can send a request to an API. If I can see your API key in the code I can grief your program and abuse your credentials to hit that quota. As a result of hitting that quota, I render your service useless bc you can't use that API anymore. 

The canonical way to handle this problem is to have a hidden `.env` file somewhere that stores all your API keys and other credentials. You'd have your script read from this `.env` file and store your credentials in some constant so nobody can know your API key.